### DISCUSSION TITLE
Tried to use supertest but getting error

### DISCUSSION THREAD

**Discussion by diego-rapoport** [TypedProject/ts-express-decorators#1794] [Q&A] [answered]
The error:

      process.exit called with "1"
      at importFiles (node_modules/@tsed/components-scan/src/importFiles.ts:19:17)
      at node_modules/@tsed/components-scan/src/importComponents.ts:52:25
          at async Promise.all (index 1)
      at importComponents (node_modules/@tsed/components-scan/src/importComponents.ts:59:55)
          at async Promise.all (index 2)
      at importProviders (node_modules/@tsed/common/src/utils/importProviders.ts:29:22)
      at PlatformBuilder.importProviders (node_modules/@tsed/common/src/builder/PlatformBuilder.ts:323:23)
      at PlatformBuilder.runLifecycle (node_modules/@tsed/common/src/builder/PlatformBuilder.ts:182:5)
      at before (node_modules/@tsed/common/src/services/PlatformTest.ts:56:18)

The `spec` file
```import { PlatformTest } from "@tsed/common";
import SuperTest from "supertest"
import { Server } from "../src/Server";

describe("Server", () => {

  let request: SuperTest.SuperTest<SuperTest.Test>;

  beforeAll(PlatformTest.bootstrap(Server))
  beforeAll(() => {
    request = SuperTest(PlatformTest.callback())
  })

  it("should call GET /api/usuarios", async () => {
    const response = await request.get("/api/usuarios");
    console.log(response)
    expect(response).toBe(true)
  });
});

**Comment by Romakita** [comment]
Ok create a reproducible example. With your example, it's impossible to help...
And the error doesn't help

**Reply by Romakita** [reply]
In this case, please issue. There is a template to fill correctly the issue and give the minimal required information.

**Comment by diego-rapoport** [comment]
Sorry, don't really know how to make a reproducible example. Any suggestions? I'm new to this framework and I'm really enjoying it, but I lack experience specially with typescript.

**Reply by Romakita** [reply]
You can share me your repo or create a small repository example with the CLI.

**Reply by Romakita** [reply]
because there is no reason to not works. So the issue is probably somewhere else in your project.

**Reply by diego-rapoport** [reply]
Will do in a few minutes.

**Comment by diego-rapoport** [comment, marked as answer]
https://github.com/diego-rapoport/tsEdBackend

**Reply by Romakita** [reply]
Sorry for the very late answers. March and April was complicated for me.

I found your issue. Is about your import => `src/modelos/cliente.model` in cliente.service.ts. You have to replace by `../modelos/cliente.model`. 

Please don't use absolute url. In major case, it cause issue in node.js. It works if you are in pure javascript project. But with typescript is not a good idea to use it. If you build your project, all paths with src will be wrong.

So fix your paths ;). I hope you have found the problem since your last message!

See you
